scylla-cql/src/frame/value.rs

```diff
@@ -364,6 +364,22 @@ impl std::hash::Hash for CqlVarint {
     }
 }
 
+/// Native CQL `decimal` representation.
+///
+/// Represented as a pair:
+/// - a [`CqlVarint`] value
+/// - 64-bit integer which determines the position of the decimal point
```

Comment:
The native representation (i.e. how Scylla/Cassandra transfers data over the socket) uses `i32` here.